src/patterns/cookies-page/full-page/index.njk

```diff
@@ -0,0 +1,77 @@
+---
+title: Full page – Cookies page
+layout: layout-example-part-page.njk
+---
+
+{% extends "example-wrappers/full-page.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "Prototype"
+    },
+    html: "This isn't a real service – it is an example."
+  }) }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Cookies</h1>
+      <p class="govuk-body">Cookies are small files saved on your phone, tablet or computer when you visit a website.</p>
+      <p class="govuk-body">We use cookies to make this site work and collect information about how you use our service.</p>
+      <h2 class="govuk-heading-m">Essential cookies</h2>
+      <p class="govuk-body">Essential cookies keep your information secure while you use this service. We do not need to ask permission to use them.</p>
+      {{ govukTable({
+        caption: "Essential cookies we use",
+        captionClasses: "govuk-table__caption--s",
+        firstCellIsHeader: false,
+        head: [
+            {
+            text: "Name"
+            },
+            {
+            text: "Purpose"
+            },
+            {
+            text: "Expires"
+            }
+        ],
+        rows: [
+            [
+            {
+                text: "session_cookie"
+            },
+            {
+                text: "Used to keep you signed in"
+            },
+            {
+                text: "20 hours"
+            }
+            ],
+            [
+            {
+                text: "cookie_policy"
+            },
+            {
+                text: "Saves your cookie consent settings"
+            },
+            {
+                text: "1 year"
+            }
+            ]
+        ]
+        }) }}
+      <h2 class="govuk-heading-m">Analytics cookies (optional)</h2>
+      <p class="govuk-body">With your permission, we use Google Analytics to collect data about how you use this service. This information helps us to improve our service.</p>
+      <p class="govuk-body">Google is not allowed to use or share our analytics data with anyone.</p>
+      <p class="govuk-body">Google Analytics stores anonymised information about:</p>
```

Comment:
Since this heading's incomplete, maybe we can just delete?

Reply:
Yeah i'm up for that, I'll push up a change soon

Reply:
Done!